app/src/main/res/layout/topic_practice_fragment.xml

```diff
@@ -13,7 +13,7 @@
     android:id="@+id/topic_practice_skill_list"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:background="@color/oppia_background"
+    android:background="@color/topic_activity_layout_background_color"
```

Comment:
For this and everywhere else, suggest creating new colors like `topic_practice_fragment_background_color`. Considering this color name contains `activity` keyword it can create confusion in future.

Reply:
Done that